app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -1,5 +1,7 @@
 package io.pumpkinz.pumpkinreader.model;
 
+import android.util.Log;
```

Comment:
Useful tip: make code formatter also optimize imports.
So this one right here will be removed automatically when you format the code